SmartTutor/ContentModel/AssessmentEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SmartTutor.ContentModel
+{
+    using KnowledgeComponentModel.KnowledgeComponents;
+
+    public abstract class AssessmentEvent
```

Comment:
Maybe we should have this be an interface (with getKC as a method). Not sure though, we can leave it as is for now.